dependencies.yaml

```diff
@@ -341,7 +341,7 @@ dependencies:
           - sphinx-markdown-tables
           - sphinx<6
           - sphinxcontrib-websupport
-          - &pylibcugraphops pylibcugraphops==23.12.*
+          - pylibcugraphops==23.12.*
```

Comment:
Wouldn't it be better to keep the anchor and use the alias elsewhere instead of repeating the same version pin, or is there a reason the anchor shouldn't be used? This is undoing the change made in [this PR](https://github.com/rapidsai/cugraph/pull/3898/files#diff-5475a6d76de4c506ee92cf6f941bba30a6a07b5881cfea531d42a6ec035095a6), was that intended?

Reply:
Good catch, this should actually be done the same way as [other libs](https://github.com/rapidsai/cugraph/blob/branch-23.10/dependencies.yaml#L515-L538) that are both conda packages and suffixed pip packages.